phoenix-core/src/main/java/org/apache/phoenix/coprocessor/UngroupedAggregateRegionObserver.java

```diff
@@ -202,7 +207,10 @@ protected RegionScanner doPostScannerOpen(final ObserverContext<RegionCoprocesso
             ScanUtil.setRowKeyOffset(scan, offsetToBe);
         }
         final int offset = offsetToBe;
-
+        if(MetaDataUtil.hasLocalIndexColumnFamily(region.getTableDesc())) {
+        	nonLocalIndexFamilies = MetaDataUtil.getNonLocalIndexColumnFamilies(region.getTableDesc());
+        }
+        
```

Comment:
This is not required. Removed it. Just while debugging one of the issue added it.